api/src/main/java/org/pmiops/workbench/billing/NotificationService.java

```diff
@@ -1,7 +1,14 @@
 package org.pmiops.workbench.billing;
 
+import java.time.LocalDate;
 import org.pmiops.workbench.db.model.DbUser;
 
 public interface NotificationService {
-  void alertUser(DbUser user, String msg);
+  void alertUserFreeTierDollarThreshold(
```

Comment:
These parameter lists are intended to match the email templates in https://docs.google.com/document/d/1v8v0bhPMLUMwpO9B8uXmYoymNSncF4uf_n7i0RzwT54/edit#heading=h.b5vt06vb56bp